src/Team-Services-Bot.Api/Cards/LogOnCard.cs

```diff
@@ -0,0 +1,50 @@
+﻿// ———————————————————————————————
+// <copyright file="LogOnCard.cs">
+// Licensed under the MIT License. See License.txt in the project root for license information.
+// </copyright>
+// <summary>
+// Contains the login card.
+// </summary>
+// ———————————————————————————————
+
+namespace Vsar.TSBot.Cards
+{
+    using System;
+    using System.Collections.Generic;
+    using Microsoft.Bot.Connector;
+    using Resources;
+
+    /// <summary>
+    /// Represent the login card.
+    /// </summary>
+    public class LogOnCard : SigninCard
+    {
+        private const string Scope = "vso.agentpools%20vso.build_execute%20vso.chat_write%20vso.code%20vso.connected_server%20" +
```

Comment:
I am not 100% sure, but isn't this string popping up in a few places? If so we should have a global constant.